src/core/modules/Planet.cpp

```diff
@@ -916,15 +916,17 @@ class SolarEclipse
 		double L1 = z * tf1 + (0.2725076 / cos(f1));
 		double L2 = z * tf2 - (0.272281 / cos(f2));
 		double mu = gast - a / M_PI_180;
-		constexpr double e2 = 0.00669438;
+		const double f = 1. - ssystem->getEarth()->getOneMinusOblateness(); // flattening
```

Comment:
good to identify these as well. could become static, and save a multiplication with 
`e2=f*(2.-f);
`